src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -12,92 +12,177 @@ This page quickly describes how [Issues](https://guides.github.com/features/issu
 
-- Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers and Wording Managers.
-- Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts and QA automation engineers.
-- Developers team a.k.a Core Team: this team takes care of the code and act as maintainer for the project.
+- Core Product Team: this team takes care of the product vision of the software. It has Product Managers, UX Designers, and a Product Content Manager (employees of PrestaShop's company)
+- Quality Assurance (QA) team: this team takes care of ensuring the quality of the software. It has QA analysts, QA automation engineers (employees of PrestaShop's company) and issues managers.
+- Core Maintainers Team: this team takes care of the code and act as maintainer for the project (most of them are employees of the PrestaShop Company, but not all of them)
 
 A maintainer will naturally belong to the Developers team but can also assist and work with the Product Team and QA Team.
 
 ## Issues
 
 Currently, there are 4 different types of Issues submitted by users on GitHub.
 
-### Feature Request
+### Feature request or improvement
 
-When a user submits a Feature Request, it can be either a Product feature request (example: add a new Back office page) or a Technical feature request (example: support PostgreSQL).
+When a user submits a feature request or an improvement, it can be either a functional request to add or change a behavior (example: add a new Back office page) or a technical request (example: support PostgreSQL).
 
-Product feature requests are labelled "Waiting for PM" while Technical feature request are labelled "Waiting for dev".
+Functional requests are labelled "Waiting for PM" while technical requests are labelled "Waiting for dev" by the issue manager (belonging to the QA team) in charge of qualifying the issues.
 
-The team in charge will then analyse the request and either reject it or accept it. Rejected feature requests are closed while accepted feature requests are added to the roadmap and labelled. Labels will help triaging the backlog.
+The Core Product team or the Core Maintainers team in charge will then analyse the request and either reject it or accept it. Rejected requests are closed with "no change required" label, while accepted requests are added to the backlog and labelled as "to be specified" if researches or specifications are needed or "to do". Labels will help triaging the backlog.
 
 ### Bug report
 
-When a user submits a bug report, the QA team will analyse it and attempt to reproduce it. If it can be reproduced and is confirmed to be an issue, it will be labelled and added to the roadmap.
+When a user submits a bug report, the issue manager will analyse it and attempt to reproduce it. If it can be reproduced and is confirmed to be an issue, it will be labelled as "to do" and added to the backlog.
```

Comment:
Who is the issue manager ? 
Maybe you could add link to the dedicated page. 

http://doc.prestashop.com/pages/viewpage.action?spaceKey=PSRD&title=Issue+Manager+Cheat+Sheet#IssueManagerCheatSheet-TheIssueManagerjob

Reply:
Thank you for your feedback @ldeprestashop

I just updated the first part with the role of the issue manager which belong the QA team
I didn't link the documentation because it's only accessible for PS employees for now, we will need to move it soon so that it will be accessible to everyone :)

Reply:
We could keep somewhere a list of "who is who" but I dont think this should be in a ... developer documentation 😅  . Even this page is not really at the right place. We are not writing documentation here 😛  .